include/riak_cs_gc_d.hrl

```diff
@@ -106,3 +93,14 @@
 -define(DEFAULT_GC_BATCH_SIZE, 1000).
 -define(DEFAULT_GC_WORKERS, 2).
 -define(EPOCH_START, <<"0">>).
+
+-record(gc_manager_state, {
```

Comment:
This state record is referenced only from manager module, I guess.
Maybe just historical(?) reason, how about putting it into manger module file?